cell_classification/evaluation_script.py

```diff
@@ -17,51 +15,56 @@
         "--model_path",
         type=str,
         help="Path to model weights",
-        default="C:\\Users\\lorenz\\Desktop\\angelo_lab\\cell_classification\\checkpoints\\" +
-        "ex_2\\ex_2.h5",
+        default=None,
     )
     parser.add_argument(
         "--params_path",
         type=str,
         help="Path to model params",
-        default="C:\\Users\\lorenz\\Desktop\\angelo_lab\\cell_classification\\checkpoints\\"
-        + "ex_2\\params.toml",
+        default="E:\\angelo_lab\\test\\params.toml",
+    )
+    parser.add_argument(
+        "--worst_n",
```

Comment:
I think having an option for `best_n` would be helpful as well to enable easy comparisons